test/cpp/interop/client_helper.cc

```diff
@@ -107,14 +97,10 @@ std::shared_ptr<Channel> CreateChannelForTestCase(
   snprintf(host_port, host_port_buf_size, "%s:%d", FLAGS_server_host.c_str(),
            FLAGS_server_port);
 
-  if (test_case == "service_account_creds") {
```

Comment:
https://github.com/grpc/grpc/blob/master/test/cpp/interop/client.cc#L71 and
the following references to service_account_creds also need to be removed :)

On Thu, Aug 27, 2015 at 3:52 PM, jboeuf <notifications@github.com> wrote:

> In test/cpp/interop/client_helper.cc
> <https://github.com/grpc/grpc/pull/3075#discussion_r38155823>:
>
> > @@ -107,14 +97,10 @@ std::shared_ptr<Channel> CreateChannelForTestCase(
> >    snprintf(host_port, host_port_buf_size, "%s:%d", FLAGS_server_host.c_str(),
> >             FLAGS_server_port);
> >
> > -  if (test_case == "service_account_creds") {
>
> Done.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/grpc/grpc/pull/3075/files#r38155823>.
>